### DISCUSSION TITLE
[Question][Hw1] question about code coverage and warning

### DISCUSSION THREAD

**Discussion by githubqookie** [prosyslab-classroom/cs424-program-reasoning#288] [Q&A]
Name: Kim Seung Gi

1. I made example3 to improve code coverage, and I copied and pasted the code in test/dune and just changed the number(example1 -> example3). But some error appears when I use "make coverge". Is there any way to fix this error?

2. There is no error or warning in the code itself, but when I use "make" command, some warning appears because there is an unused value. Is it ok to have these kind of warnings? ("make release" command doesn't reject the code)

below image contains both question
<img width="631" alt="스크린샷 2023-09-10 오후 9 58 06" src="https://github.com/prosyslab-classroom/cs492-program-reasoning/assets/143948838/99d1ada1-2932-43ca-9e2a-9faa88a57203">

p.s Because of question 1, I just changed the example1.c to improve code coverage. Is this fine too?
thank you for reading.

**Comment by goodtaeeun** [comment]
1. Refer to #145 to add a custom test case.
2. You can ignore the warnings in `make`. Just check for warnings generated with `make release`.

**Comment by githubqookie** [comment]
Thank you for answering!
